wire-ios-data-model/Source/Model/Conversation/ZMConversation+MLS.swift

```diff
@@ -206,6 +202,41 @@ public extension ZMConversation {
     }
 }
 
+// MARK: - Migration releated fetch requests
+
+public extension ZMConversation {
+
+    static func fetchAllTeamGroupProteusConversations(in context: NSManagedObjectContext) throws -> [ZMConversation] {
+        let selfUser = ZMUser.selfUser(in: context)
+        guard let selfUserTeamIdentifier = selfUser.teamIdentifier else {
+            assertionFailure("selfUser.teamIdentifier == nil")
+            return []
+        }
+
+        let request = NSFetchRequest<Self>(entityName: Self.entityName())
+        request.predicate = NSCompoundPredicate(
+            andPredicateWithSubpredicates: [
+                .hasConversationType(.group),
+                .hasMessageProtocol(.proteus),
```

Comment:
Since I'm planning to write a similar request, I would suggest adding the protocol as a parameter in the method so it can be changed, based on which protocol we would like to use. In my case: It's `mixed`.

Reply:
done: 1727fe505d6e78e2ff431b775e6a77f145e9cf4f